rest.rst

```diff
@@ -246,7 +245,7 @@ Input
 + ``appName`` (Required): Name of your Testmunk app.
 + ``file`` (Required): iOS or apk app file. Only the format .ipa and .apk allowed.
 + ``testcases`` (Required): Zip file containing the features folder. Zip file should contain the zipped features folder, as you would upload to our website.
-+ ``email`` (Optional): An email to this address will be sent after the testrun has been successfully executed. Needs to be an email address of an existing testmunk user.
```

Comment:
*An email to this address will be sent after the testrun has been successfully executed*

The thing is that it's not related. We should rather say that it will be send to every person which is on the list in Notification view on the dashboard